qa/die-with-dignity/src/javaRestTest/java/org/elasticsearch/qa/die_with_dignity/DieWithDignityIT.java

```diff
@@ -62,7 +62,7 @@ public void testDieWithDignity() throws Exception {
                     ".*ElasticsearchUncaughtExceptionHandler.*",
                     ".*javaRestTest-0.*",
                     ".*fatal error in thread \\[Thread-\\d+\\], exiting.*",
-                    ".*java.lang.OutOfMemoryError: die with dignity.*"
+                    ".*java.lang.OutOfMemoryError: Requested array size exceeds VM limit.*"
```

Comment:
This leaks jvm implementation into our tests. I am not really against that, but perhaps we can add a comment to remove the text from matching if it fails?